osu.Game/Graphics/Backgrounds/Triangles.cs

```diff
@@ -153,7 +153,7 @@ protected override void Update()
                 TriangleParticle newParticle = parts[i];
 
                 // Scale moved distance by the size of the triangle. Smaller triangles should move more slowly.
```

Comment:
Seems like this comment is no longer valid (the second portion specifically). Not sure how to reword it though.

Reply:
It's still valid? The only different is it's clamped slightly now.

Reply:
Hmm, got more unsure with that "more slowly", but yeah nevermind.